plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/quickfix/AddFunctionToSupertypeFix.kt

```diff
@@ -82,6 +83,15 @@ class AddFunctionToSupertypeFix private constructor(
         }
     }
 
+    private fun KtNamedFunction.removeDefaultValueIfNeeded() {
+        if (hasModifier(KtTokens.OVERRIDE_KEYWORD)) {
```

Comment:
Why do we need `hasModifier(KtTokens.OVERRIDE_KEYWORD)` check? Isn't it always true?